addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2171,7 +2171,7 @@ const SnippetOptionWidget = Widget.extend({
 
         function applyCSS(cssProp, cssValue, styles) {
             if (!weUtils.areCssValuesEqual(styles[cssProp], cssValue)) {
-                this.$target[0].style.setProperty(cssProp, cssValue, 'important');
+                this.$target[0].style.setProperty(cssProp, cssValue);
```

Comment:
> For the removal of !important on inline CSS, if it was really needed (to override stylesheet with background-color: !important) maybe I could add a data-html4-compatible attribute and only not add !important when it's set.

Annoying indeed... I guess that could be a solution. We cannot just remove !important always, it is indeed ... important ... to override some stylesheet style and in general our editor panel should be able to override any custom style coming from anywhere, that's why we use important. Note that we could have coded a solution which tries to set the style as not important, check if it had an effect, then try to add as important otherwise (we already have code trying out classes to see if it has an effect or not in this same function). Could be a cool improvement... that could be considered stable.